MeetHut.Backend/Middlewares/DatabaseMigration.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace MeetHut.Backend.Middlewares 
+{
+    
+    /// <summary>
+    /// Database Migration
+    /// </summary>
+    public static class DatabaseMigration 
```

Comment:
Maybe migration by configuration. `appsettings.json` and `appsettings.Development.json`

Reply:
I will check it.